src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/JourneyPriceCatalogueController.kt

```diff
@@ -27,30 +24,32 @@ private val logger = loggerFor<JourneyPriceCatalogueController>()
  * Controller responsible for generating and returning an on demand journey price catalogue spreadsheet to the end user.
  */
 @RestController
-@SessionAttributes(SUPPLIER_ATTRIBUTE)
 class JourneyPriceCatalogueController(
   private val journeyPriceCatalogueService: JourneyPriceCatalogueService,
   private val timeSource: TimeSource
 ) {
 
-  @GetMapping("/generate-prices-spreadsheet/{supplier}")
```

Comment:
This URI has been changed sightly.  The supplier is actually on the session so we do not need to pass this through.